classes/order/OrderSlip.php

```diff
@@ -330,8 +330,9 @@ public static function create(Order $order, $product_list, $shipping_cost = fals
 
             $address = Address::initialize($order->id_address_invoice, false);
             $id_address = (int) $address->id;
-            $id_tax_rules_group = Product::getIdTaxRulesGroupByIdProduct((int) $order_detail->product_id);
-            $tax_calculator = TaxManagerFactory::getManager($address, $id_tax_rules_group)->getTaxCalculator();
+            $id_tax_rules_group = (int) $order_detail->id_tax_rules_group;
+            $id_order_detail = (int) $product['id_order_detail'];
```

Comment:
This could be simplified using `$order->getTaxCalculator()` which does the same thing, the `$order_detail->id_tax_rules_group;` part is correct however

Reply:
Hey @jolelievre,
I cannot find a function called getTaxCalculator() in Order class, so maybe could you explain your suggestion a little bit?

And regardings the test: I would be happy if you could implement them, I'm not so familiar with this topic.

Reply:
My bad I meant `$order_detail->getTaxCalculator()`

Reply:
And no problem I will add the behat tests ;)

Reply:
Oh ok, this makes sense, I added the change.